ui-ngx/src/app/shared/components/file-input.component.scss

```diff
@@ -83,6 +83,12 @@ $previewSize: 100px !default;
       }
     }
   }
+
+  .input-hint {
+    cursor:pointer;
+    transform: translate(5px,-3px);
+    position: absolute;
+  }
```

Comment:
Why did you change?? I could not find this class in the component.

Reply:
forget to add class 
lost put class on html after marge 
QA sked to level hint